tests/unit/Hooks/ImageHooksTest.php

```diff
@@ -0,0 +1,224 @@
+<?php
+/**
+ * ownCloud - mediametadata
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Jalpreet Singh Nanda (:imjalpreet) <jalpreetnanda@gmail.com>
+ * @copyright Jalpreet Singh Nanda (:imjalpreet) 2016
+ */
+
+namespace OCA\MediaMetadata\Hooks;
+
+function getimagesize($filename = null, array &$imageinfo = null) {
+	return ImageHooksTest::$dimensions ?: getimagesize($filename);
```

Comment:
You don't need the check here.

1) we want it to always return the static dimensions
2) the filename will never exist